### PR TITLE
Added Prototype for Activity Feed

### PR DESCRIPTION
**Pull request by lealgamb**
I created the activity feed servlet and it's corresponding jsp file. I also updated web.xml to map to the corresponding url. Currently, the activity feed page doesn't fetch any data and only has the doGet() method implemented. Here's a screenshot:

image

### CODE REVIEW

File: src/main/webapp/WEB-INF/view/activityfeed.jsp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <a href="/profile/<%=request.getSession().getAttribute("user") %>">
+         Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else{ %>
+      <a href="/login">Login</a>
+    <% } %>
+    <a href="/about.jsp">About</a>
+  </nav>
+
+  <div id="container">
+  	<h1>Activity</h1>
```
Comment (yebrahim) [inline src/main/webapp/WEB-INF/view/activityfeed.jsp]: nitpick: tabs vs spaces ;)

File: src/main/java/codeu/controller/ActivityFeedServlet.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package codeu.controller;
+
+import java.io.IOException;
```
Comment (yebrahim) [inline src/main/java/codeu/controller/ActivityFeedServlet.java]: As @vocatwk mentioned, I'd go over these to make sure nothing is being imported unnecessarily.

**Comment by jasminechau2** [comment]
It looks good! Keep on working on it!

File: src/main/java/codeu/controller/ActivityFeedServlet.java (hunk 2 above)
Comment (lealgamb) [reply src/main/java/codeu/controller/ActivityFeedServlet.java]: Will do, thank you!

**Pull request merged**